packages/react-urql/src/provider.tsx

```diff
@@ -0,0 +1,123 @@
+import { createClient as createWSClient } from 'graphql-ws'
+import React, { PropsWithChildren } from 'react'
+import {
+  cacheExchange,
+  createClient as createUrqlClient,
+  dedupExchange,
+  Exchange,
+  fetchExchange,
+  Provider as UrqlProvider,
+  RequestPolicy,
+  subscriptionExchange
+} from 'urql'
+import { refocusExchange } from '@urql/exchange-refocus'
+import { devtoolsExchange } from '@urql/devtools'
+
+import type { NhostClient } from '@nhost/nhost-js'
+
+export type NhostUrqlClientOptions = {
+  nhost?: NhostClient
+  graphqlUrl?: string
+  headers?: {
+    [header: string]: string
+  }
+  requestPolicy?: RequestPolicy
+  exchanges?: Exchange[]
```

Comment:
This is unused. I think we should merge exchanges provided by external users with the internal list.

Reply:
fixed